test/integration/other/form-entities-version.js

```diff
@@ -436,19 +437,20 @@ describe('Update / migrate entities-version within form', () => {
         .expect(200);
 
       // check updatedAt on the draft form
-      await asAlice.get('/v1/projects/1/forms/updateEntity')
+      await asAlice.get('/v1/projects/1/forms/updateEntity/draft')
         .expect(200)
         .then(({ body }) => {
           should(body.updatedAt).be.null();
         });
 
-      const { acteeId } = await Forms.getByProjectAndXmlFormId(1, 'updateEntity').then(o => o.get());
+      const { acteeId } = await Forms.getByProjectAndXmlFormId(1, 'updateEntity', Form.WithoutDef).then(o => o.get());
       await Audits.log(null, 'upgrade.process.form.entities_version', { acteeId });
 
       // Run form upgrade
       await exhaust(container);
 
-      await asAlice.get('/v1/projects/1/forms/updateEntity')
+      // REVIEW maybe this shouldn't change - maybe the base form endpoint should always provide basic info
+      await asAlice.get('/v1/projects/1/forms/updateEntity/draft')
```

Comment:
Same theme as some of my other comments -- Yeah I don't think this needs to change. Base form endpoint always providing basic info makes sense. That was the main thing I was concerned about with an API change "possibly breaking users' workflows".